.editorconfig

```diff
@@ -9,7 +9,7 @@ indent_size = 4
 max_line_length = 120 # Same as .clang-format
 
 [*.py]
-max_line_length = 88 # Same as Black
+max_line_length = 88 # Same as Ruff's default
```

Comment:
If you'd like to change that to have a consistent line-length across all files. I can change it in `.clang-format` and/or `ruff.toml` to your preference in an follow-up PR.

Reply:
> If you'd like to have a consistent line-length across all files. 

God no - that's something I wish these linting tools cared less about :)